public/app/plugins/datasource/grafana-azure-monitor-datasource/query_ctrl.ts

```diff
@@ -1,7 +1,6 @@
 import { defaultsDeep, find, map, isString } from 'lodash';
 import { QueryCtrl } from 'app/plugins/sdk';
 import TimegrainConverter from './time_grain_converter';
-import './editor/editor_component';
```

Comment:
Is this file used? I believe you should be able to remove this and the annotations_query_ctrl.

Reply:
Yup - feel free to remove query_ctrl and annotations_query_ctrl also @iwysiu !

Reply:
And the [partials](https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/prometheus/partials/) too. :)